### PR TITLE
WIP - 6 workers aws configuration

### PR DESCRIPTION
**Pull request by AviadP**
this PR created for bug verification only and shell not be merged or reivew

Signed-off-by: aviadp <apolak@redhat.com>

### CODE REVIEW

File: conf/deployment/aws/ipi_3az_rhcos_3m_6w.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+ENV_DATA:
+  worker_replicas: 6
+  worker_instance_type: 'm5.4xlarge'
```
Comment (petr-balogh) [inline conf/deployment/aws/ipi_3az_rhcos_3m_6w.yaml:5]: If this is going to be production config I would like to keep all the details like we have here: https://github.com/red-hat-storage/ocs-ci/blob/6fab465c910f88a999c81a8eb271c73ba26b11d5/conf/deployment/aws/ipi_3az_rhcos_3m_3w.yaml just increase number of nodes + nodes which suppose to be labeled to 6 as well.

File: conf/deployment/aws/ipi_3az_rhcos_3m_6w.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    - 'us-east-2b'
+    - 'us-east-2c'
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 6
+  master_replicas: 3
+  worker_instance_type: 'm5.4xlarge'
+  ocs_operator_nodes_to_label: 6
```
Comment (petr-balogh) [inline conf/deployment/aws/ipi_3az_rhcos_3m_6w.yaml:20]: I guess this is not right section and should be only on line 4 in DEPLOYMENT section.

Comment (petr-balogh) [reply conf/deployment/aws/ipi_3az_rhcos_3m_6w.yaml:20]: Add # TODO: Add deployment polarion ID

**Comment by AviadP** [comment]
closing this PR

**Pull request closed without merging**